bitnami/zookeeper/templates/_helpers.tpl

```diff
@@ -261,3 +265,55 @@ zookeeper: tls.quorum.enabled
     enable auto-generated certificates.
 {{- end -}}
 {{- end -}}
+
+{{/*
+Validate values of ZooKeeper - Quorum TLS Auth
+*/}}
+{{- define "zookeeper.validateValues.quorum.tls.auth" -}}
+{{- if and .Values.tls.quorum.enabled .Values.tls.quorum.auth }}
+{{- $tlsAuths := list "none" "want" "need" -}}
+{{- if not (has .Values.tls.quorum.auth $tlsAuths) }}
+zookeeper: tls.quorum.auth
+    tls.quorum.auth must be one of these values: "none", "want" or "need".
+{{- end -}}
+{{- end -}}
+{{- end -}}
```

Comment:
This is sweet, but we are already validating it in the docker image 😁 

https://github.com/bitnami/bitnami-docker-zookeeper/blob/208bb77b98f32d461637f853c6e18f6d7dd04677/3.8/debian-10/rootfs/opt/bitnami/scripts/libzookeeper.sh#L103-L104

I think that we can safely remove this provided that we have a note in the `values.yaml` file about the possible values.

Reply:
Yes, I already saw this check in the container. The only difference by checking it in helm too is that the chart will fail at applying instead of failing when the container will try to boot. Thus, we can detect the misconfiguration sooner. But yes, the result is the same, an error will be caught. I can remove it if it is redundant :)

Reply:
To me, we are safe by documenting the possible values in the `values.yaml`/`README.md` and performing the check in the container. After all, the container will show the validation error when trying to boot, so this validation is kind of redundant IMHO :)

Reply:
yes, it makes sense! I removed the check